exercises/list-ops/example.go

```diff
@@ -40,22 +40,7 @@ func (s IntList) Filter(fn predFunc) IntList {
 }
 
 // Length returns the length of a list
-func (s IntList) Length() int {
-	// for fun, let's do this with recursion
-	var lengthAcc func(IntList, int) int
-	// yeah, this is kind of cheaty, but it's hard to determine if a slice is
-	// empty without using len, so walling it off
-	isEmpty := func(l IntList) bool {
-		return len(l) == 0
-	}
-	lengthAcc = func(lst IntList, acc int) int {
-		if isEmpty(lst) {
-			return acc
-		}
-		return lengthAcc(lst[1:], acc+1)
-	}
-	return lengthAcc(s, 0)
-}
+func (s IntList) Length() int { return len(s) }
```

Comment:
Isn't the point of this exercise NOT to use already existing functions?

Note: I'm not a big fan of such exercises especially since it's not clear what `existing functions` includes.

Reply:
In general, yes. I get your point.

If somebody wants to learn, they need to implement it by themselves. However, example solution should be idiomatic go. When somebody has a slice, using something else than `len` to find length of this slice will never be seen in the wild. That's why it's fair to make this change. 

That being said, if you have any other suggestions to improve this, I am happy to update.

Reply:
I'd probably vote for removing this exercise altogether 😄 

But to answer your question: Changes are fine with me.

Reply:
Same. 

To move on here, merging then.